way_to_home/static/src/components/loginForm/loginForm.js

```diff
@@ -38,18 +44,16 @@ class LoginForm extends Component {
 
     onClickConfirm = () => {
         let type = this.state.request_type;
-
+        let self = this;
         axios.post('http://127.0.0.1:8000/api/v1/user/' + type, {
             email: this.state.email,
             password: this.state.first_pass,
-            //save_cookie: this.state.save_cookie,
         })
-            .then(function (response) {
-                console.log(response);
-                this.props.close();
+            .then(() => {
+                window.location.reload();
```

Comment:
This is a bad way to reload a page because it reloads all files, DOM, scripts. Better solution is React `withRouter()`

Reply:
And maybe you should somehow tell a user that he was logged in?

Reply:
How it can be in the best way?

Reply:
> How it can be in the best way?
`You about React withRouter()`

Reply:
I think you should add a state with success and error keys in `Home` component, and handlers for them. Move these handlers to your component with props. Now you can show `<CustomizedSnackbars>` in Home component, I think `<Map>`  component will also use it